src/fsharp/PackageManager.Integration.fs

```diff
@@ -38,15 +38,17 @@ type PackageManager = {
     ToolName: string
     Name:string }
 
-let RegisteredPackageManagers : PackageManager list = 
+let RegisteredPackageManagers = lazy (
```

Comment:
What is the goal of lazy?

I understand we want to have this list being built dynamically eventually, and that it should work even in case we add possibility to add handlers dynamically (and also with loading more scripts), will lazy allow this kind of thing to happen?